test/integration/CMakeLists.txt

```diff
@@ -58,7 +58,7 @@ set(tests
   world_dom.cc
 )
 
-if (PYTHONINTERP_FOUND AND PY_PSUTIL)
+if (Python3_Interpreter_FOUND AND PY_PSUTIL)
```

Comment:
This seems out of place. I see that [92d4e9c](https://github.com/ignitionrobotics/sdformat/pull/1008/commits/92d4e9c26d98d1403fb64c3c7c875e4cab50365e) has been included in this PR, but that PR should already be on `main` as [8a3e5a8](https://github.com/ignitionrobotics/sdformat/commit/8a3e5a828241405516e6601d353ed7b0993c1b8b)🤔

Reply:
Oops

https://github.com/ignitionrobotics/sdformat/pull/1008/commits/b751698896d4c9d87e09d6cd09d8dcff5e54b5ad